assets/scss/modules/_navigation.scss

```diff
@@ -147,3 +145,145 @@ a.back-to-top-link {
   }
 
 }
+
+/*
+Page Navigation
+
+A page navigation element with a previous and next link.
+
+Markup:
+<nav class="page-nav" role="navigation" aria-label="Page navigation">
+  <ul>
+    <li class="page-nav__item page-nav__item--previous">
+      <a class="page-nav__link page-nav__link-previous" title="Navigate to previous part" rel="prev" href="#">
+        <span class="page-nav__label">Previous </span>
+        <span class="page-nav__title">Overview</span>
+      </a>
+    </li>
+    <li class="page-nav__item page-nav__item--next">
+      <a class="page-nav__link page-nav__link-next" title="Navigate to next part" rel="next" href="#">
+        <span class="page-nav__label">Next</span>
+        <span class="page-nav__title">Inheritance Tax reliefs</span>
+      </a>
+    </li>
+  </ul>
+</nav>
+
+Styleguide Navigation.Page Navigation
+*/
```

Comment:
@feedmypixel perhaps `Previous/Next` gives a better expectation of what to expect before landing on the page?